src/sections/Playground/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+import Reviews from "../Pricing/review-slider";
+import PlaygroundHero from "./playground-hero";
+import PlaygroundFeatures from "./playground-features";
+
+const PlaygroundWrapper = styled.div`
+
+`;
```

Comment:
If it's empty then why do you need to create it at the first place?
Just use empty tags for enclosing the components like
```
<>
     <PlaygroundHero />
     <PlaygroundFeatures />
     <Reviews />
</>
```

Reply:
@Nikhil-Ladha I did this initially since I had the impression that I'll be needing this going further on the page. But since it is of no use currently, I'll remove the wrapper. Thanks for pointing this out.